resources/kcp/charts/provisioner/values.yaml

```diff
@@ -23,7 +24,7 @@ gardener:
   maintenanceWindowConfigMapName: ""
   secretName: "gardener-credentials"
   auditLogsPolicyConfigMap: ""
-  manageSecrets: true
+  manageSecrets: false 
```

Comment:
Please don't modify either the chart template, or the chart default values. We should provide an override in our config repo.